### DISCUSSION TITLE
How can I render each row as a card?

### DISCUSSION THREAD

**Discussion by sahmed007** [icflorescu/mantine-datatable#569] [Q&A]
I'd like to apply some sort of styling on each row so that it looks like a grid of cards rather than a table. How can I do this?

**Comment by icflorescu** [comment]
Why would you want to use Mantine DataTable instead of simple [Mantine cards](https://mantine.dev/core/card/) for this?...

**Reply by sahmed007** [reply]
I'm tasked with creating an app which will have about 10 columns per record. Maybe about 50k records. Will there be a performance hit if I choose those Mantine cards? I was suggested to use the datatable.
